### PR TITLE
osdocs-667 GCP docs updates

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://jira.coreos.com/browse/OSDOCS-667

@patrickdillon, will you PTAL?

### CODE REVIEW

File: modules/installation-gcp-service-account.adoc [hunk 1]
```diff
@@ -6,24 +6,29 @@
...
 .Procedure
 
-. Create a new service account in the project that you use to host your
+. Create a service account in the project that you use to host your
 {product-title} cluster. See
 link:https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating_a_service_account[Creating a service account]
 in the GCP documentation.
 
 . Grant the service account the appropriate permissions. You can either
 grant the individual permissions that follow or assign the `Owner` role to it.
```
Comment (patrickdillon) [inline modules/installation-gcp-service-account.adoc]: > the individual permissions that follow

Seems to be missing the list of needed permissions, although I am an adocs newb and perhaps the list is composed from elsewhere.

**Comment by patrickdillon** [comment]
@kalexand-rh These changes LGTM.

**Comment by kalexand-rh** [comment]
@wmengRH, will you PTAL?

@openshift/team-documentation, will you please review?

**Comment by ahardin-rh** [comment]
LGTM!

**Comment by wmengRH** [comment]
LGTM

**Comment by kalexand-rh** [comment]
Thank you all! Merging.

**Pull request merged**